src/components/Card.js

```diff
@@ -234,3 +216,10 @@ export const Card = ({ direction = 'ltr' }) => {
     </div>
   );
 };
+
+function setEncryptionMode(network, e) {
```

Comment:
Convert to an ES6 function
and no need to pass network as an input parameter , as its already global and we will need it always
```JS
const setEncryptionMode = e => {
    ...network,
    encryptionMode: e.target.value,
  };
```

Or you could call setNetwork here
```JS
const setEncryptionMode = e => setNetwork({
    ...network,
    encryptionMode: e.target.value,
  });
```
Then you can change this

```diff
- onChange={(e) => setNetwork(setEncryptionMode(network, e))}
+ onChange={setEncryption}
```

Reply:
Thank you! I improved the code and now it is ES6.